examples/slider/slider-color-viewer.html

```diff
@@ -0,0 +1,422 @@
+                    <circle class="background" cx="14" cy="24" r="12"/>
+                    <line x1="11" y1="24" x2="17" y2="24"/>
+                  </svg>
+                </button>
+                <div class="color-slider"
+                    role="slider"
+                    tabindex="0"
+                    aria-valuemin="0"
+                    aria-valuenow="128"
+                    aria-valuemax="255"
+                    aria-labelledby="id-blue">
+                  <svg width="290" height="38">
+                    <text class="value" x="125" y="12">128</text>
+                    <rect class="hc-background" x="4" y="16" rx="6" width="279" height="20" fill="white"/>
+                    <rect class="rail" x="6" y="18" rx="4" width="275" height="16"/>
+                    <rect class="fill blue" x="6" y="19" rx="4" width= "128" height="14" />
+                    <rect class="thumb" x="132" y="18" rx="4"  width="16" height="16" />
+                  </svg>
+                </div>
+                <button class="change inc" aria-label="increase blue one" tabindex="-1">
+                  <svg width="32" height="38">
+                    <circle class="hc-background" cx="14" cy="24" r="14"/>
+                    <circle class="background" cx="14" cy="24" r="12"/>
+                    <line x1="10" y1="24" x2="18" y2="24"/>
+                    <line x1="14" y1="20" x2="14" y2="28"/>
+                  </svg>
+                </button>
+                <button class="change inc10" aria-label="increase blue ten" tabindex="-1">
+                  <svg width="32" height="38">
+                    <circle class="hc-background" cx="14" cy="24" r="14"/>
+                    <circle class="background" cx="14" cy="24" r="12"/>
+                    <line x1="5" y1="24" x2="13" y2="24"/>
+                    <line x1="9" y1="20" x2="9" y2="28"/>
+                    <line x1="15" y1="24" x2="23" y2="24"/>
+                    <line x1="19" y1="20" x2="19" y2="28"/>
+                  </svg>
+                </button>
+              </div>
+
+              <h3>Color View Box</h3>
+              <div class="color-info">
+                <div class="color-box"></div>
+                <label>
+                  Color (HEX):
+                  <input type="text" readonly class="color-value-hex" value="">
+                </label>
+                <label>
+                  Color (RGB):
+                  <input type="text" readonly class="color-value-rgb" value="">
+                </label>
+              </div>
+            </div>
+          </div>
+          <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+  <section>
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>
+          The three sliders are wrapped in a group to help assistive technology users understand that all three elements together represent a color picker.
```

Comment:
Did you want to say "viewer" here?

```suggestion
          The three sliders are wrapped in a group to help assistive technology users understand that all three elements together represent a color viewer.
```